support-frontend/assets/helpers/internationalisation/Readme.md

```diff
@@ -2,9 +2,9 @@
 
 The internationalisation helper is formed by:
 
--   CountryGroup
--   Country
--   Currency
+- CountryGroup
+- Country
+- Currency
```

Comment:
According to the git history, these spaces were originally added by Prettier (commit [79d38103c015](https://github.com/guardian/support-frontend/commit/79d38103c015d086a8af5c85204a427286ab5091)) – is the change now due to a prettier config change that wasn’t applied to these files?